backend/dataall/core/environment/cdk/pivot_role_core_policies/service_quota.py

```diff
@@ -0,0 +1,22 @@
+from dataall.core.environment.cdk.pivot_role_stack import PivotRoleStatementSet
+from aws_cdk import aws_iam as iam
+
+
+class ServiceQuotaPivotRole(PivotRoleStatementSet):
+    """
+    Class including all permissions needed  by the pivot role to work with AWS Service Quota.
+    It allows pivot role to:
+    - List and Get Service Quota details
+    """
+
+    def get_statements(self):
+        statements = [
+            # Service Quota - Needed to determine the number of service quotas for managed policies which can be attached
+            iam.PolicyStatement(
+                sid='ServiceQuotaListGet',
+                effect=iam.Effect.ALLOW,
+                actions=['servicequotas:List*', 'servicequotas:Get*'],
+                resources=['*'],
```

Comment:
I confirm that we cannot restrict this policy to specific resources. It requires a wildcard

Reply:
Hmm.. Can you tell me how to go about finding this ? 
Also, I do not get any exception thrown for this at all 😅  . Maybe because I am getting service quotas for iam and there is a `iam:Get*` permission somewhere

Reply:
Exactly, we must be granting iam:Get somewhere (which is fine, it is read only) This list comes from granting the permissions manually to any role in the AWS Console. It provides hints on the types of resources that can be used in a policy and other associated policies that are needed